initialise/initialise.go

```diff
@@ -51,9 +51,17 @@ func (e *ExternalServiceList) GetConsumer(ctx context.Context, cfg *config.Confi
 		Offset:       &kafkaOffset,
 		KafkaVersion: &cfg.KafkaVersion,
 	}
+	if cfg.KafkaSecProtocol == "TLS" {
+		cgConfig.SecurityConfig = kafka.GetSecurityConfig(
```

Comment:
Do you need to check if these params exist and are valid if this config property is set?

Reply:
This is a great idea and is complete now